File: engine/model/src/main/scala/net/server/channel/packet/cash/use/UseNotePacket.scala
package net.server.channel.packet.cash.use

class UseNotePacket(private var _position: Short, private var _itemId: Int, private var _to: String, private var _message: String) extends AbstractUseCashItemPacket(_position, _itemId) {
  def to: String = _to

  def message: String = _message
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/use/UseNotePacket.java
package net.server.channel.packet.cash.use;

public class UseNotePacket extends AbstractUseCashItemPacket {
   private final String to;

   private final String message;

   public UseNotePacket(Short position, Integer itemId, String to, String message) {
      super(position, itemId);
      this.to = to;
      this.message = message;
   }

   public String to() {
      return to;
   }

   public String message() {
      return message;
   }
}
